modules/accounts.js

```diff
@@ -206,46 +211,6 @@ Accounts.prototype.isLoaded = function () {
  * @see {@link http://apidocjs.com/}
  */
 Accounts.prototype.shared = {
-
-	getBalance: function (req, cb) {
-		library.schema.validate(req.body, schema.getBalance, function (err) {
-			if (err) {
-				return setImmediate(cb, err[0].message);
-			}
-
-			self.getAccount({ address: req.body.address }, function (err, account) {
-				if (err) {
-					return setImmediate(cb, err);
-				}
-
-				var balance = account ? account.balance : '0';
-				var unconfirmedBalance = account ? account.u_balance : '0';
-
-				return setImmediate(cb, null, {balance: balance, unconfirmedBalance: unconfirmedBalance});
-			});
-		});
-	},
-
-	getPublickey: function (req, cb) {
-		library.schema.validate(req.body, schema.getPublicKey, function (err) {
-			if (err) {
-				return setImmediate(cb, err[0].message);
-			}
-
-			self.getAccount({ address: req.body.address }, function (err, account) {
-				if (err) {
-					return setImmediate(cb, err);
-				}
-
-				if (!account || !account.publicKey) {
-					return setImmediate(cb, 'Account not found');
-				}
-
-				return setImmediate(cb, null, {publicKey: account.publicKey});
-			});
-		});
-	},
-
 	getDelegates: function (req, cb) {
```

Comment:
Please remove the functionality as it is not present in the new API endpoints for accounts